metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -483,7 +475,7 @@ private ApiError createTopic(CreatableTopic topic,
                         " times: " + e.getMessage());
             }
         }
-        Uuid topicId = new Uuid(random.nextLong(), random.nextLong());
+        Uuid topicId = Uuid.randomUuid();
```

Comment:
Note that we have switched from a regular `Random` to a `SecureRandom`. Probably a good idea, but it could have perf implications.

Reply:
Yeah, I'm aware. But it felt like something that had to be done. The rate of topic creation is typically not high anyway.

Reply:
Can we note this in the PR description?